cmd/controller/main.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"github.com/tektoncd/pipeline/pkg/clock"
```

Comment:
@vdemeester @lbernick  We used to have our own `system.Clock` in knative.dev/pkg for exactly this reason and we moved to `k8s.io/apimachinery/pkg/util/clock` instead.

Reply:
It looks like this could be a drop-in replacement here.

Reply:
👍🏼